packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -197,7 +197,7 @@ const HelperText = createComponentSlot(FormHelperText)<FieldContainerClassKey>({
 export const FieldContainer = (inProps: React.PropsWithChildren<FieldContainerProps>) => {
     const {
         variant = "vertical",
-        fullWidth,
+        fullWidth: passedFullWidth,
```

Comment:
I think you could just set the default value here:

```suggestion
        fullWidth: variant === "horizontal",
```

Reply:
That doesn't work because `variant` is defined in the same destructuring, so it's not available yet